src/serviceDesks/genesys/genesysServiceDesk.ts

```diff
@@ -31,22 +31,27 @@ import {
   LOG_PREFIX,
   MessageEvent,
 } from './genesysTypes';
-import { setup } from './purecloudSetup';
+import {
+  ORGANIZATION_ID,
+  DEPLOYMENT_ID,
+  QUEUE_TARGET,
+  AUTH_SERVER_BASE_URL,
+} from '../../middleware/genesys/src/config/constants';
 import { messages } from './stockMessages';
 
 /**
  * Indicates whether your chat deployment requires authentication.
  * If true, this code will attempt to fetch a memberAuthToken from the authentication server pointed to
- * by AUTH_SERVER_BASE_URL in '../../purecloudSetup.ts' and use it when creating the chat.
+ * by AUTH_SERVER_BASE_URL in '../../middleware/genesys/src/config/constants.ts' and use it when creating the chat.
  */
-const WIDGET_REQUIRES_AUTHENTICATION = false;
+const WIDGET_REQUIRES_AUTHENTICATION = true;
```

Comment:
`WIDGET_REQUIRES_AUTHENTICATION` keep this to false, user can change it according to his needs